### ISSUE TITLE
Dependency alert: kramdown version

### ISSUE THREAD

**Issue by SarahAlidoost** [eWaterCycle/ewatercycle.github.io#20]
GitHub found potential security vulnerabilities in the dependencies, see the message [here](https://github.com/eWaterCycle/ewatercycle.github.io/security/dependabot/Gemfile.lock/kramdown/open) and [Gemfile.lock](https://github.com/eWaterCycle/ewatercycle.github.io/blob/master/Gemfile.lock).

**Comment by bvreede** [comment]
Thanks Sarah! The message link does not work, but when running `bundle install` without a lock file present, a new lock file is created that contains a higher version of kramdown.
Would that fix the issue?
See also PR #21.

**Comment by SarahAlidoost** [comment]
Thanks @bvreede for fixing the issue. Here I copied the message, hope that it helps.
```
Dependabot cannot update kramdown to a non-vulnerable version
The latest possible version that can be installed is 1.17.0 because of the following conflicting dependencies:

github-pages (193) requires kramdown (= 1.17.0)
github-pages (193) requires kramdown (~> 1.14) via jekyll (3.7.4)
The earliest fixed version is 2.3.1.
```

**2 kramdown vulnerabilities found in Gemfile.lock on Aug 8, 2020**
Remediation
Upgrade kramdown to version 2.3.1 or later. For example:

`gem "kramdown", ">= 2.3.1"`
Always verify the validity and compatibility of suggestions with your codebase.

Details
CVE-2021-28834
Vulnerable versions: >= 1.16.0, < 2.3.1
Patched version: 2.3.1
Kramdown before 2.3.1 does not restrict Rouge formatters to the Rouge::Formatters namespace, and thus arbitrary classes can be instantiated.

CVE-2020-14001
Vulnerable versions: < 2.3.0
Patched version: 2.3.0
The kramdown gem before 2.3.0 for Ruby processes the template option inside Kramdown documents by default, which allows unintended read access (such as template="/etc/passwd") or unintended embedded Ruby code execution (such as a string that begins with template="string://<%= `). NOTE: kramdown is used in Jekyll, GitLab Pages, GitHub Pages, and Thredded Forum.

**Comment by bvreede** [comment]
That is useful, @SarahAlidoost, thanks! It seems the kramdown risks have been patched in 2.3.1, and the lock file I generated uses kramdown 2.3.0. Locally, this is generated when running `build install`, and I'm finding repeated recommendations to not include the lockfile in the repository, except in the context of a jekyll website, but even that [is not entirely clear](https://stackoverflow.com/questions/56261127/is-the-gemfile-lock-file-needed-in-a-jekyll-site-hosted-with-github-pages).

I'm wondering whether we should not just remove the lockfile, but also whether that will solve the issue. The lockfile for #21  is generated with https://rubygems.org/gems/github-pages, and kramdown 2.3.0 is listed there; unless this version is updated, any lockfile generated will have kramdown 2.3.0.

Kiiiinda just want to try it out, but I also don't want to break the site 😇

**Comment by bvreede** [comment]
Oh and adding gem `"kramdown", ">= 2.3.1"` to the Gemfile only crashes bundler with error message:
```
Bundler could not find compatible versions for gem "kramdown":
  In Gemfile:
    kramdown (>= 2.3.1)

    github-pages was resolved to 8, which depends on
      kramdown (= 1.0.2)
```
Weirdly, without that line, kramdown is never 1.0.2 in any reference to it 🤷🏼‍♀️
